source/adapter.js

```diff
@@ -1,7 +1,12 @@
 // Adapter's interface.
-var AdapterJS = AdapterJS || {};
+window.AdapterJS = window.AdapterJS || {};
```

Comment:
@letchoo @serrynaimo 
Is this change necessary? 
If we change the namespacing of AJS I think we need to move to 0.11.0. anyway